estilo/palettes/night-owl.vim.yml

```diff
@@ -37,3 +37,6 @@ gray7: '#777777'
 graya: '#aaaaaa'
 purple2: '#2d2c5d'
 hoki: '#5f7e97'
+
+tags: '#5A455B'
```

Comment:
seems this is not used

Reply:
yes, was experimenting, this can be deleted.